.github/workflows/ci.yml

```diff
@@ -25,25 +25,25 @@ jobs:
       - name: Install NPM
         run: |
           npm install
-          
+
       - name: Run NPM
         run: |
           npm run all
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          
+
   test:
     if: github.event_name  == 'pull_request'
     runs-on: ubuntu-latest
     steps:
       # the below actions use the local state of the action. please replace `./` with `mikepenz/release-changelog-builder-action@{latest-release}`
       # Showcases how to use the action without a prior checkout
       # Since 3.2.0 the configuration can be provided within the `yml` file
-      - name: "Configuration without Checkout"
+      - name: 'Configuration without Checkout'
```

Comment:
Can you please revert the changes made here?

Reply:
Of course. That's how prettier thought to format the file. 
I added an override in for prettier to use double quotes in .yml and re-formatted it. 
It's not exactly reverted, since some quotes were single quotes from before, but it should be more consistent now; I hope you agree with the change.